packages/contracts/test/helpers/LiveTestHelper.sol

```diff
@@ -1,62 +1,62 @@
 
-import "../../src/dollar/BondingV2.sol";
+import "../../src/dollar/Staking.sol";
 import "../../src/dollar/mocks/MockBondingV1.sol";
 import "../../src/dollar/mocks/MockShareV1.sol";
-import "../../src/dollar/BondingFormulas.sol";
-import "../../src/dollar/BondingShareV2.sol";
+import "../../src/dollar/StakingFormulas.sol";
+import "../../src/dollar/StakingShare.sol";
 import "../../src/dollar/interfaces/IMetaPool.sol";
-import "../../src/dollar/UbiquityGovernance.sol";
-import "../../src/dollar/UbiquityAlgorithmicDollarManager.sol";
-import "../../src/dollar/mocks/MockuADToken.sol";
+import "../../src/dollar/core/UbiquityGovernanceToken.sol";
+import "../../src/dollar/core/UbiquityDollarManager.sol";
+import "../../src/dollar/mocks/MockDollarToken.sol";
 import "../../src/dollar/UbiquityFormulas.sol";
-import "../../src/dollar/TWAPOracle.sol";
-import "../../src/dollar/MasterChefV2.sol";
-import "../../src/dollar/UARForDollarsCalculator.sol";
+import "../../src/dollar/core/TWAPOracleDollar3pool.sol";
+import "../../src/dollar/UbiquityChef.sol";
+import "../../src/dollar/core/CreditRedemptionCalculator.sol";
 import "../../src/dollar/interfaces/ICurveFactory.sol";
 import "../../src/dollar/interfaces/IMasterChef.sol";
-import "../../src/dollar/CouponsForDollarsCalculator.sol";
-import "../../src/dollar/DollarMintingCalculator.sol";
-import "../../src/dollar/mocks/MockDebtCoupon.sol";
-import "../../src/dollar/DebtCouponManager.sol";
-import "../../src/dollar/UbiquityAutoRedeem.sol";
-import "../../src/dollar/ExcessDollarsDistributor.sol";
+import "../../src/dollar/core/CreditNFTRedemptionCalculator.sol";
+import "../../src/dollar/core/DollarMintCalculator.sol";
+import "../../src/dollar/mocks/MockCreditNFT.sol";
+import "../../src/dollar/core/CreditNFTManager.sol";
+import "../../src/dollar/core/UbiquityCreditToken.sol";
+import "../../src/dollar/core/DollarMintExcess.sol";
 import "../../src/dollar/SushiSwapPool.sol";
 import "../../src/dollar/interfaces/IERC1155Ubiquity.sol";
 
-import "Uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
-import "Uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
 
 import "forge-std/Test.sol";
 
 contract LiveTestHelper is Test {
     using stdStorage for StdStorage;
 
-    Bonding bondingV1;
-    BondingV2 bondingV2;
-    BondingFormulas bFormulas;
-    BondingShareV2 bondingShareV2;
-
-    UbiquityAlgorithmicDollarManager manager;
-
-    UbiquityFormulas uFormulas;
-    TWAPOracle twapOracle;
-    MasterChefV2 chefV2;
-    UARForDollarsCalculator uarCalc;
-    CouponsForDollarsCalculator couponCalc;
-    DollarMintingCalculator dollarMintCalc;
-    MockDebtCoupon debtCoupon;
-    DebtCouponManager debtCouponMgr;
-    UbiquityAutoRedeem uAR;
-    ExcessDollarsDistributor excessDollarsDistributor;
+    Bonding stakingV1;
```

Comment:
Do we need versioning? I expect that we don't.

Reply:
this is intentional

how it was:
```
Bonding bondingV1;
BondingV2 bondingV2;
```

how it is now:
```
Bonding stakingV1;
Staking staking;
```

so this is for new developers to understand that now we use `Staking staking`, but earlier we used `Bonding stakingV1;` and that "bonding" is the same thing as "staking"

this is complicated nevetherless)